app/src/main/java/com/babylon/wallet/android/presentation/onboarding/restore/mnemonics/RestoreMnemonicsScreen.kt

```diff
@@ -74,15 +74,26 @@ fun RestoreMnemonicsScreen(
         state = state,
         onBackClick = viewModel::onBackClick,
         onSkipClicked = viewModel::onSkipClick,
+        onSkipMainSeedPhraseClicked = viewModel::onSkipMainSeedPhraseClicked,
```

Comment:
let's not use the past tense and keep the same as the android has "Click" 
that's also for the rest of the team if you all agree 
@micbakos-rdx @jakub-rdx